documentation/documentation/i18n/nl/docusaurus-plugin-content-docs/current/projectform/project_upload_form.md

```diff
@@ -0,0 +1,30 @@
+# Project upload handleiding
+Een leraar kan een nieuw project van zijn cursus uploaden door naar de url `projects/create` te gaan, waar hij gegevens moet invullen die overeenkomen met het project dat hij wil uploaden.
+
+![Projectformulier](/img/project_form_1.png)
+
+De eerste 3 velden zijn vrij eenvoudig, je hoeft alleen maar de titel in te vullen die je voor het project wilt, de beschrijving en de cursus te selecteren waarvoor je het project wilt. De cursusselectie bevindt zich in een vervolgkeuzemenu waarin je alle cursussen ziet waarvan je leraar bent.
+
+![Projectformulier](/img/project_form_2.png)
+
+Vervolgens is er een kalender waar je de deadlines van het project moet kiezen, elke deadline kan een deadline en een tijdstempel hebben. Zoals te zien is in de afbeelding hierboven, zie je een tabel van elke deadline die je hebt toegevoegd, als je een deadline wilt verwijderen, klik dan gewoon op de datum in de kalender en klik op het verwijdersymbool.
```

Comment:
Same as the English file, how can a deadline have a deadline, maybe make this a bit clearer

Reply:
die je hebt toegevoegd, als... -> die je hebt toegevoegd,. Als...

Reply:
https://github.com/SELab-2/UGent-3/pull/250/commits/c2af2565e6508f3e8caa588f821307480a79c078